### ISSUE TITLE
High Contrast Theme color options behave inconsistently

### ISSUE THREAD

**Issue by mauro1855** [highcharts/highcharts#19683]
Hi all,

I've notice that highContrastTheme options seem to work for some types of chart series but not for others. For example, as far as I can tell, the highContrastTheme options to define series color work correctly for 'column' series (the user-defined high contrast color is correctly applied), but not for 'line' series (the user-defined high contrast color is ignored and the default series color is used) and yet again in a completely different way for 'pareto' series (it ignores any color configuration and uses 'windowText' color in high contrast mode).

#### Expected behaviour
colors defined highContrastTheme should be correctly renderer in the chart when in High Contrast Mode

#### Actual behaviour
highContrastTheme-defined colors don't work for some series types. Even the way it doesn't work is not consistent across series. Specifically I found that:
- Line series ignore the color specified in highContrastTheme and use the color defined for the series.
- Pareto series ignore any defined color and use 'windowText'. Tooltip also shows incorrect color!

#### Live demo with steps to reproduce
For Line series: https://jsfiddle.net/mauro1855/o3y47g2h/2/
For column + pareto series: https://jsfiddle.net/9pft8weL/

1. Open the fiddles and observe the configuration and the rendered result.
2. Enable HighContrastMode in Windows; or use Chrome's rendering emulation of forced colors.
3. Re-render chart (or refresh JS Fiddle).
4. Observe the colors of the series. Do you see how they differ from the configured colors in highContrastTheme?
5. Bonus for Pareto in HighContrastMode: do you see how the tooltip color does not match the pareto line color?

#### Product version
See JS fiddle. I believe it is happening in the latest version.

#### Affected browser(s)
At least in Chrome, haven't checked others.

**Comment by oysteinmoseng** [comment]
Thanks for reporting!

Internal note: We should consider this together with https://github.com/highcharts/highcharts/issues/15921 when it comes to how we deal with forced colors.
